### PR TITLE
Added documentation for swagger page

### PR DESCRIPTION
**Pull request by Marishka17**

### CODE REVIEW

**Comment by bsekachev** [comment]
@nmanovic 

Maria is our student from NNSTU

**Comment by bsekachev** [comment]
@Marishka17 

Can you please write unresolved issues in comments to your PR?

**Comment by Marishka17** [comment]
Unresolved issues:
- request(load_data_proxy) is not displayed on swagger page

File: cvat/apps/engine/views.py [hunk 1]
```diff
@@ -565,6 +687,21 @@ def annotations(self, request, pk):
                     return Response(data=str(e), status=status.HTTP_400_BAD_REQUEST)
                 return Response(data)
 
+class NoPagingFilterAutoSchema(SwaggerAutoSchema):
```
Comment (bsekachev) [inline cvat/apps/engine/views.py]: @Marishka17 
Please remove this class. It was my bad, users actually are paginated.

**Comment by Marishka17** [comment]
Type commands below into the terminal window to generate a typescript client according to Swagger API specification.

	git clone https://github.com/swagger-api/swagger-codegen
	cd swagger-codegen
	./run-in-docker.sh mvn package

	wget http://localhost:7000/api/swagger.json --header "Authorization: Token ${your-token}" -O modules/swagger.json
	./run-in-docker.sh generate -i modules/swagger.json -l typescript-fetch -o /gen/out/typescript-cvat -DpackageName=cvat

The generated client part will be located in ./out/typesckipt-cvat/ 

	- api.ts
	- configuration.ts
	- custom.d.ts
	- git_push.sh
	- index.ts
	- .gitignore
	- .swagger-codegen/VERSION
	- swagger-codegen-ignore

**Comment by bsekachev** [comment]
@Marishka17 

Please, fix the codacy issue: "Unused import"

**Comment by nmanovic** [comment]
@Marishka17 ,

Welcome to our team! Great job! Please give me a couple of days to try the PR.

**Pull request merged**